### DISCUSSION TITLE
~/.cache/trivy/db/trivy.db

### DISCUSSION THREAD

**Discussion by yanivpaz** [aquasecurity/trivy#1944] [Q&A] [answered]
Hi

how .cache/trivy/db/trivy.db is being maintained and generated  ?
what is the CVE DB which is being used   ?

Thanks
Yaniv

**Comment by itaysk** [comment, marked as answer]
Hi, you can see the source for so called trivy-db here: https://github.com/aquasecurity/trivy-db
